include/tvm/relay/function.h

```diff
@@ -144,6 +144,8 @@ constexpr const char* kComposite = "Composite";
 constexpr const char* kInline = "Inline";
 /*! \brief Indicate the function was created by the Pattern Partitioning Pass. */
 constexpr const char* kPartitionedFromPattern = "PartitionedFromPattern";
+/*! \brief Indicate the target that the function should be lowered to. */
+constexpr const char* kTarget = "Target";
```

Comment:
The other "target" attr is used in https://github.com/apache/tvm/blob/main/include/tvm/ir/function.h#L170 for tir functions, would be great if we can consolidate